cypress/README.md

```diff
@@ -44,6 +44,7 @@ Use scenario tags:
 5. `@ignore` and `@[bug-number]` should be removed after bug fix
 6. `@build` for pipeline - each build is verified by those tests
 7. `@accessibility` tests verifies accessibility violations
+8. `@validations` for validation component
```

Comment:
we should describe tags in this way -> `tests verify validation components`

Reply:
fixed